### ISSUE TITLE
Fix keyboard a11y of UserMenu panel

### ISSUE THREAD

**Issue by dananjohnson** [ManifoldScholar/manifold#2194]
- [ ] focus should go to first link in panel when opened
- [ ] using `tab` and `shift + tab` should land focus on next/previous item or close panel as appropriate

**Comment by dananjohnson** [comment]
Currently when the panel is opened, focus goes to a "Close Panel" button that's SR-only (part of the `UIPanel` component). This could be moved to the end of the panel, or optionally removed if a prop is passed. But would that improve accessibility?

For the second to-do, closing the panel on `tab` or `shift + tab` goes against the focus trap we're using here. It's not clear to me that implementing this, rather than closing the panel using `ESC` or the "Close Panel" button, is an improvement.

@zdavis any thoughts?

**Comment by zdavis** [comment]
We discussed and want to review in detail with UW. The user has to actively press space to open this menu, so we don't understand why they need to tab or shift+tab out of it.

**Comment by avaaec** [comment]
Ava from UW here, same comment as isssue #2187, could you give more detailed directions on where to find this panel and/or what specific feedback we gave that prompted this issue? Thanks!

**Comment by dananjohnson** [comment]
@avaaec this is based on issue 30 filled by your team:
> Improper implementation of listbox. Upon opening the profile button, focus should go to the first list item in the listbox. Also, pressing tab or shift+tab should close the listbox and place focus on the next/previous item from the listbox, respectively

The panel in question is this (we assume), which can be accessed from the banner when you are logged in:
<img width="329" alt="Screen Shot 2019-06-24 at 9 42 12 AM" src="https://user-images.githubusercontent.com/12454203/60036576-63657400-9664-11e9-8f51-424488d97ac5.png">

**Comment by zdavis** [comment]
@avaaec When you have a free minute or two, we'd love some guidance on this issue. Thanks!

**Comment by avaaec** [comment]
Hi same, same response as issue #2187, I've put off responding to this issue because I was trying to clarify with a former colleague as to why it was reported in the first place. After reviewing it, I can't see any reason why this panel should be implemented as a drop down list box. It's current keyboard behavior is fine.

**Comment by zdavis** [comment]
Thanks!
